### PR TITLE
add error code 451 - https://tools.ietf.org/html/rfc7725#section-3

### PR DESCRIPTION
**Pull request by Graziel**
https://tools.ietf.org/html/rfc7725#section-3

### CODE REVIEW

File: src/Network/Exception/UnavailableForLegalReasonsException.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    /**
+     * Constructor
+     *
+     * @param string|null $message If no message is given 'Unavailable For Legal Reasons' will be the message
+     * @param int $code Status code, defaults to 451
+     */
+    public function __construct($message = null, $code = 451)
+    {
+        if (empty($message)) {
```
Comment (dereuromark) [inline src/Network/Exception/UnavailableForLegalReasonsException.php]: Can we use === null check here?

Comment (Graziel) [inline src/Network/Exception/UnavailableForLegalReasonsException.php]: its a copy paste from
https://github.com/cakephp/cakephp/blob/master/src/Network/Exception/UnauthorizedException.php
just changed default message, other exceptions also use same template : )

**Comment by markstory** [comment]
Seems reasonable to me.

**Comment by lorenzo** [comment]
Thanks!

**Pull request merged**